clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -6612,6 +6612,38 @@ the configuration (without a prefix: ``Auto``).
      int a [5];                    vs.      int a[5];
      int a [5][5];                 vs.      int a[5][5];
 
+.. _SpaceInComments:
+
+**SpaceInComments** (``Struct``) :versionbadge:`clang-format 21` :ref:`¶ <SpaceInComments>`
```

Comment:
This is not handwritten, but generated by `clang/doc/tools/dump_format_style.py`, you need to make your documentation in `Format.h`.